src/dsp_tools/utils/connection_live.py

```diff
@@ -133,6 +204,7 @@ def _write_request_to_file(
             "headers": headers,
             "params": params,
             "body": json.loads(jsondata) if jsondata else None,
+            "filepath": filepath,
```

Comment:
same here: "filepath" doesn't tell me enough. "uploaded file" or so would be more telling.